package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sensei-lms",
-  "version": "4.19.1",
+  "version": "4.19.2",
```

Comment:
I think we need to run an `npm install` to update the other version in the package lock.